libs/novui/src/recipes/button.recipe.ts

```diff
@@ -134,9 +138,10 @@ export const BUTTON_RECIPE = defineSlotRecipe({
       },
       transparent: {
         root: {
-          border: 'none',
-          bg: '[transparent]',
-          px: '0',
+          border: 'none !important',
+          borderRadius: '0 !important',
+          bg: '[transparent !important]',
+          px: '0 !important',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Why are these `!important`s needed? Can we remove?‏

Reply:
Thanks! This I think was a mistake after rebasing onto the `css-layers` branch